torch/export/export_schema.py

```diff
@@ -0,0 +1,302 @@
+from dataclasses import dataclass
+from enum import auto, Enum
+from typing import List, Union, Dict
+
+
+################################################################################
+# Following section is the defining the permissible argument types for operators
+
+# Copied from torchgen/model.py
+# !!! Consider if we want to add aliased dtypes, e.g float == float32. (Sherlock: No!)
```

Comment:
Although I wouldn't add aliases, I WOULD suggest switching to modernized names, either short form like b8, i8, u8 or longer form `float32` etc.

Reply:
OK. Updated to short form.